package.json

```diff
@@ -4,7 +4,7 @@
 	"publisher": "victorgz",
 	"description": "Stay organized, easily identify your default org and prevent unwanted deployments with Salesforce ColORG. The extension will automatically change the color of your VScode workspace based on the Salesforce org you're currently connected to. Define the set of rules, the colors to apply and start boosting your productivity!",
 	"icon": "assets/icon.png",
-	"version": "1.1.1",
+	"version": "1.2.0",
```

Comment:
@jlawrence124 can you please unstage this change? after the PR is merged, I've a process to bump the release and update all the relevant files (package.json, package-lock.json and changelog) automatically based on the commit type (feat, fix...)

(my bad, this part is not really well documented in the contribution section 😅)

Reply:
done https://github.com/victorgz/vscode-salesforce-colorg/pull/4/commits/480e63c9371274fe3d50f389840bee11c4190c1e